salt/beacons/watchdog.py

```diff
@@ -0,0 +1,213 @@
+# -*- coding: utf-8 -*-
+'''
+watchdog beacon
+
+Watch files and translate the changes into salt events
+
+:depends:   - watchdog Python module >= 0.8.3
+
+'''
+# Import Python libs
+from __future__ import absolute_import
```

Comment:
`print_function` and `unicode_literals` needs to be included in the import line as well.